numpy/lib/format.py

```diff
@@ -162,7 +169,7 @@
 import warnings
 from numpy.lib.utils import safe_eval
 from numpy.compat import (
-    asbytes, asstr, isfileobj, long, os_fspath, pickle
+    isfileobj, long, os_fspath, pickle
```

Comment:
I think I'd argue that every occurrence of `asbytes` and `asstr` is probably a bug, especially once we drop py2

Reply:
Doesn't backport.

Reply:
There is a reason we are putting off those cleanups.